Vineyard/Vineyard/Group.swift

```diff
@@ -14,14 +14,19 @@ class Group: Identifiable, Codable {
     var people: [String]
     var resolutions: [Resolution] = []
     var deadline: Date
+    //
+    var scoreGoal: Int
+    var currScore: Int
     
-    init(name: String, groupGoal: String, people: [String], resolutions: [Resolution] = [], deadline: Date) {
+    init(name: String, groupGoal: String, people: [String], resolutions: [Resolution] = [], deadline: Date, scoreGoal: Int) {
         self.id = UUID().uuidString
         self.name = name
         self.groupGoal = groupGoal
         self.people = people
         self.resolutions = resolutions
         self.deadline = deadline
+        self.scoreGoal = scoreGoal;
+        self.currScore = 3
```

Comment:
Just curious, any reason for this to be 3, not 0?

Reply:
:D wanted to just test if the progress bar was working